Flexa-Agents/Large-deposits/src/agent.ts

```diff
@@ -0,0 +1,108 @@
+import {
+  Finding,
+  TransactionEvent,
+  FindingSeverity,
+  FindingType,
+  getEthersProvider,
+  ethers,
+} from "forta-agent";
+import { BigNumber, utils, providers } from "ethers";
+import abi from "./abi";
+
+const AMOUNT_THRESHOLD = utils.parseEther("1").mul(1e6); // 1 million
+const AMP_TOKEN: string = "0xfF20817765cB7f73d4bde2e66e067E58D11095C2";
+const FLEXA_CONTRACT: string = "0x706D7F8B3445D8Dfc790C524E3990ef014e7C578";
+
+export const createFinding = (
+  amount: BigNumber,
+  partition: string,
+  operator: string,
+  from: string,
+  destinationPartition: string,
+  to: string,
+  operatorData: string,
+): Finding => {
+  return Finding.fromObject({
+    name: "Large Deposit",
+    description: "Large Deposit into staking pool",
+    alertId: "FLEXA-2",
+    severity: FindingSeverity.Info,
+    type: FindingType.Info,
+    metadata: {
+      value: amount.toString(),
+      fromPartition: partition.toLowerCase(),
+      operator: operator.toLowerCase(),
+      from: from.toLowerCase(),
+      destinationPartition: destinationPartition.toLowerCase(),
+      to: to.toLowerCase(),
+      operatorData: operatorData.toLowerCase(),
```

Comment:
The metadata described here in `agent.ts` is different to the metadata described in `README.md`.
See [here](https://github.com/NethermindEth/Forta-Agents/blob/548867fb5911110157e606f3c304d12bc99c9126/Flexa-Agents/Large-deposits/README.md?plain=1#L18-L24).